docs/framework/migration-guide/retargeting/vbnet_no_longer_supports_partial_namespace_qualification_for_systemwindows_apis.md

```diff
@@ -0,0 +1,10 @@
+### VB.NET no longer supports partial namespace qualification for System.Windows APIs
+
+|   |   |
+|---|---|
+|Details|Beginning in .NET 4.5.2, VB.NET projects cannot specify System.Windows APIs with partially-qualified namespaces. For example, referring to <code>Windows.Forms.DialogResult</code> will fail. Instead, code must refer to the fully qualified name (<xref:System.Windows.Forms.DialogResult>) or import the specific namespace and refer simply to <xref:System.Windows.Forms.DialogResult?displayProperty=name>.|
```

Comment:
I want it rendered as such. In the original document, it appears as `DialogResult`. https://github.com/Microsoft/dotnet-apiport/blob/master/docs/BreakingChanges/127%20VB.NET%20no%20longer%20supports%20partial%20namespace%20qualification%20for%20System.Windows%20APIs.md